js/functional/palindrome.js

```diff
@@ -7,7 +7,16 @@ import { parseString } from "../helper.js";
  */
 function isPalindromeUsingReverse(value) {
   // write your code here
-  return true;
+  value = parseString(value);
+
+  // Remove non-alphanumeric characters and convert to lowercase
```

Comment:
I see a kinda lot of comments similar to this. Actually, you don't need to write this comment again when the code itself is understood well. Adding more comments like this will make your code seems too complicated to understand. Please be careful and also check my review for the other file.